qa/automation/CareerPortal.Tests.Scenarios/Burlaka.feature

```diff
@@ -0,0 +1,119 @@
+Given user opened "career.quantori.com"  
+When user scrolls to the bottom of "career.quantori.com"  page 
+Then following "<text-containing element>" with "<text>" text should be displayed in the footer: 
+|text-containing element                                            |text                                                                    | 
+|link                                                               |Data Privacy Policy                                                     | 
+|link                                                               |Cookies Settings                                                        | 
+|copyright text                                                     |Copyright� 2021 Quantori. Custom Software Solutions. All rights reserved|
+
+Scenario Outline:1.2.3_02 Footer "logo-containing" elements should be visible at the bottom of the "career.quantori.com" page
+Given user opened "career.quantori.com"  
+When user scrolls to the bottom of "career.quantori.com" page
+Then following "<logo-containing element>" with "<logo>" logo should be displayed in the footer:
+|logo-containing element|logo                  | 
+|link button            |"Planet"              | 
+|link button            |hh                    | 
+|link button            |f                     | 
+|link button            |"Camera"              |
+|link button            |vk                    |
+|link button            |in                    |
+|scroll-to-top button   |Quantori              |
+|mailto button          |"Envelope"            | 
+
+Scenario:1.2.3_03 Footer link "Data Privacy policy" provides access to a readable and downloadable document
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+When user clicks a link with "Data Privacy policy" text in the footer
+Then a pdf file "Quantori Privacy Policy" should be displayed in the new window
+And user should be able to download a "Quantori Privacy Policy" pdf file
+
+Scenario:1.2.3_04 Footer link "Cookies Settings" opens a pop-up window with cookie settings
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+When user clicks a link with "Cookies Settings" text in the footer
+Then a pop-up window with the following text should be displayed in the same window:
+"""
+We use cookies to enhance the functionality of the website. 
+Some cookies are necessary for our site to work while others are optional to aid in the user experience
+For overall information about how we protect your data please review our Data Privacy Policy.
+"""
+
+Scenario:1.2.3_05 Footer link "Cookies Settings" allows a user to submit selected by default (necessary) cookies preferences
+Given user opened "career.quantori.com"  
+And user scrolled to the bottom of the "career.quantori.com" page
+And user clicked a link with "Cookies Settings" text in the footer
```

Comment:
This step is "When" (because action that leads to main case check is been done here)